radiography/src/main/java/radiography/RenderTreeString.kt

```diff
@@ -96,7 +97,7 @@ private fun StringBuilder.appendLinePrefix(
     if (parentDepth > 0) {
       append(' ')
     }
-    val lastChild = lastChildMask and (1 shl parentDepth).toLong() != 0L
+    val lastChild = lastChildMask[parentDepth]
```

Comment:
This is pretty nice though, on the other hand.

Reply:
😳 wow that's cool.